pkg/controller/nodedaemon/daemonsets.go

```diff
@@ -196,6 +197,9 @@ func MutateAggregatedSpec(
 	// service account
 	ds.Spec.Template.Spec.ServiceAccountName = common.ProvisionerServiceAccount
 
+	// priority class
+	ds.Spec.Template.Spec.PriorityClassName = common.PriorityClassName
+
```

Comment:
This breaks deployment of LSO in plain k8s environment. If possible - I would like us to keep plain k8s deployments as supported configuration. I personally use it for development and testing, but other folks might be interested for different reasons.

Can we add priority class as a environment variable in CSV and apply the priority class only if environment variable is set?

Reply:
cc @priyanka19-98